indigo/bulk_creator.py

```diff
@@ -47,6 +48,19 @@ class RowValidationFormBase(forms.Form):
     repealed_by = forms.CharField(required=False)
     taxonomy = forms.CharField(required=False)
 
+    def __init__(self, country, locality, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields['country'].choices = [(country.code, country.name)]
+        self.fields['locality'].choices = [(locality.code, locality.name)] \
+            if locality else []
+        self.fields['doctype'].choices = self.get_doctypes_for_country(country.code)
+        self.fields['subtype'].choices = [(s.abbreviation, s.name) for s in SUBTYPES]
```

Comment:
Hmm, if we load the subtypes here we're going to go to the db once for every row, which is lame.